lib/modules/datasource/flutter-version/types.ts

```diff
@@ -0,0 +1,22 @@
+interface FlutterCurrentRelease {
```

Comment:
Not sure if we need to export these interfaces if they're only used in this file?

Reply:
Please export, they are implicit used by typescript. They also don't harm, as they are removed at runtime.

Reply:
```suggestion
export interface FlutterCurrentRelease {
```